python/hsfs/engine/python.py

```diff
@@ -920,7 +944,7 @@ def get_training_data(
             #        training_dataset_obj, feature_view_obj, training_dataset_version
             #    )
             return self._apply_transformation_function(
-                training_dataset_obj.transformation_functions, df
+                feature_view_obj.transformation_functions, df
```

Comment:
`feature_view_obj.transformation_functions` has both ODT and MDT?

Reply:
No it just has MDT. ODT are only required to be run during online inference. While creation of training data and batch data. The on-demand features are read from the feature group directly without any recomputation.